server/cmd/user.go

```diff
@@ -217,6 +240,24 @@ func updateUser(cmd *cobra.Command, args []string) {
 		user.IsAdmin = userParams.admin
 	}
 
+	if userParams.maxFileSize != "" {
+		maxFileSize, err := humanize.ParseBytes(userParams.maxFileSize)
+		if err != nil {
+			fmt.Printf("Unable to parse max-file-size\n")
```

Comment:
Why not print the value here ?

Reply:
We could, but user passed it so they should know what's the value